lightning-invoice/src/payment.rs

```diff
@@ -773,25 +740,48 @@ where
 	}
 }
 
+/// Data structure used to track information about inflight HTLCs.
+pub struct InFlightHtlcs {
+	/// A map with liquidity value (in msat) keyed by a short channel id and the direction the HTLC
+	/// is traveling in. The direction boolean is determined by checking if the HTLC source's public
+	/// key is less than its destination. See [`InflightHtlcs::used_liquidity_msat`] for more
+	/// details.
+	pub map: HashMap<(u64, bool), u64>
```

Comment:
Does this need to be `pub`?

Reply:
Addressed in 2f811f39f6aec80d8c9531853c91d646d31df98f